runtime/rococo/src/lib.rs

```diff
@@ -1375,6 +1375,9 @@ construct_runtime! {
 		// Gilts pallet.
 		Gilt: pallet_gilt::{Pallet, Call, Storage, Event<T>, Config} = 38,
 
+		// State trie migration pallet, only temporary.
+		StateTrieMigration: pallet_state_trie_migration = 41,
```

Comment:
This index was chosen a bit randomly, please tell me if there is a more suitable one (the crate will not be in the runtime forever).